src/containers/BuySellCrypto/PaymentMethod/ManageWyreAccount/ManageWyrePersonalDetails.js

```diff
@@ -154,24 +155,20 @@ class ManageWyrePersonalDetails extends Component {
               textStyle={{ color: '#FFF' }}
             />
             <View>
-              <FormLabel labelStyle={styles.formLabel}>
-                    Legal Name:
-              </FormLabel>
-              <FormInput
+              <Input
+                label="Legal Name:"
+                labelStyle={styles.formLabel}
                 underlineColorAndroid={Colors.quaternaryColor}
                 onChangeText={(text) => this.setState({ name: text })}
                 value={this.state.name}
                 autoCorrect={false}
                 inputStyle={styles.formInputContainer}
               />
-              <FormValidationMessage labelStyle={styles.formValidationLabel}>
-                {this.state.errors.name}
-              </FormValidationMessage>
             </View>
             <View>
-              <FormLabel labelStyle={styles.formLabel}>
+              <Text labelStyle={styles.formLabel}>
```

Comment:
labelStyle is not a property of Text, but this doesn't matter because all these screens will be redone anyways